Automattic/wp-calypso
# Themes: Add current-theme query component

A [query component](https://github.com/Automattic/wp-calypso/blob/master/docs/our-approach-to-data.md#query-components) to replace the old-style fetcher component. Will make it very easy to determine current theme properly in theme sheets (#5285)

No visual changes

![screen shot 2016-05-13 at 11 56 20](https://cloud.githubusercontent.com/assets/7767559/15245996/c8bacec4-1901-11e6-89ba-a3b7ad0c4c08.png)

**To Test**
* Go to http://calypso.localhost:3000/design/
* choose a site
Expected: the _CURRENT THEME_ bar works as previously
Things to check:
* switching sites
* jetpack site
* `localStorage.clear();` in console then refresh
* activate a theme
